superwall/src/main/java/com/superwall/sdk/network/device/DeviceHelper.kt

```diff
@@ -386,7 +398,21 @@ class DeviceHelper(
     suspend fun getTemplateDevice(): Map<String, Any> {
         val identityInfo = factory.makeIdentityInfo()
         val aliases = listOf(identityInfo.aliasId)
-
+        val geo =
+            try {
+                withTimeout(1.minutes) {
```

Comment:
Added a timeout so we don't await forever here, but pass forward with a null. cc @yusuftor lmk if going forward with null (like it woould now if it didnt get the info yet) is ok, or if I should change it to throw/return